scrollytelling/src/primitive.tsx

```diff
@@ -468,17 +468,30 @@ function getValidAt(at: number) {
  * Parallax
  * -----------------------------------------------------------------------------------------------*/
 
-function Parallax({
-  children,
-  tween,
-}: {
-  children?: React.ReactNode;
+interface ParallaxProps {
   tween: Omit<TweenBaseDef, "to" | "from" | "fromTo"> & {
-    target?: TweenTarget;
-    movementX?: UnitValue;
-    movementY?: UnitValue;
+    target?: TweenTarget; // Optional: The target element or elements to apply the animation to.
+    movementX?: UnitValue; // Optional: The amount of movement on the X-axis.
+    movementY?: UnitValue; // Optional: The amount of movement on the Y-axis.
   };
-}) {
+  children?: React.ReactNode; // Optional: Content to be rendered inside the Parallax component.
+}
+
+/**
+ * Parallax component applies a parallax effect to its children using GSAP animations.
```

Comment:
nitpick: Would omit the component name since this message appears on hover so it's a bit redundant
```suggestion
 * Usage: Applies a parallax effect to its children using GSAP animations.
```